### PR TITLE
Support next up time limit

### PR DESCRIPTION
**Pull request by 1hitsong**
**Changes**
Create new user setting called Max Days Next Up. This value is used to set the maximum amount of days a show should stay in the Next Up list on the homepage without watching it.

This value exists on the Web UI, but was missing in Roku.

### CODE REVIEW

File: settings/settings.json [hunk 1]
```diff
@@ -31,7 +31,20 @@
...
-                "title": "Details Page",
+                "title": "Home Page",
+                "description": "Options for Home page.",
+                "children": [
+                    {
+                        "title": "Max Days Next Up",
+                        "description": "Set the maximum amount of days a show should stay in the 'Next Up' list without watching it.",
+                        "settingName": "ui.details.maxdaysnextup",
+                        "type": "integer",
+                        "default": ""
```
Comment (neilsb) [inline settings/settings.json]: Wed defaults this to 365 days.  Worth doing the same here?

Comment (1hitsong) [reply settings/settings.json]: Default of 365 days added. 

Additionally, as a sign of saving, when the user clicks the Check button we now move the cursor focus back to the left navigation.

**Review by neilsb — APPROVED** [review]

File: settings/settings.json [hunk 2]
```diff
@@ -31,7 +31,20 @@
...
+                        "title": "Max Days Next Up",
+                        "description": "Set the maximum amount of days a show should stay in the 'Next Up' list without watching it.",
+                        "settingName": "ui.details.maxdaysnextup",
+                        "type": "integer",
+                        "default": "365"
+                    }
+                ]
+            },
+            {
+                "title": "Detail Page",
```
Comment (alanazar) [inline settings/settings.json]: Corrected line was deleted above and then re-added with the old name
```suggestion
                "title": "Details Page",
```

Comment (1hitsong) [reply settings/settings.json]: This was one of the conflicts I resolved.

**Review by neilsb — APPROVED** [review]

**Pull request merged**